actions-runner-controller.yaml

```diff
@@ -18,7 +18,7 @@ pipeline:
     with:
       repository: https://github.com/actions/actions-runner-controller
       tag: gha-runner-scale-set-${{package.version}}
-      expected-commit: 8aa04dd2beddce108a759afd6bf4ea95640a270e
+      expected-commit: 3fab744a4fadf4909a8395f8f295265549464496
```

Comment:
```suggestion
      expected-commit: 3fab744a4fadf4909a8395f8f295265549464496
      
  - uses: go/bump
    with:
      deps: golang.org/x/crypto@v0.17.0
      go-version: "1.21"
```

Reply:
@debasishbsws can this be in moved to a separate PR with advisory data added